website/templates/project/view_file.mako

```diff
@@ -149,6 +149,7 @@
         file: {
             size: ${size},
             extra: ${extra},
+            sourceUrl: '${source_url}',
```

Comment:
You need to use `js_str` here
You may also want to add an if in so that it actually gets set to `null` or `undefined` as per the above comment. but to quote sloria, follow your :heart: